src/SonarScanner.MSBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -123,6 +123,10 @@ private async Task<bool> DoExecute(ProcessedArgs localSettings)
 
             var server = this.factory.CreateSonarQubeServer(localSettings);
 
+            //TODO: fail fast after release of 6.0
+            //Deprecation notice for SQ < 7.9
+            await server.WarnIfSonarQubeVersionIsDeprecated();
```

Comment:
There's not test for this?

Reply:
Not sure what you meant here ?

Reply:
`WarnIfSonarQubeVersionIsDeprecated` itself is tested well.

`TeamBuildPreProcessor` invokes that method, but there's no test to be sure what `TeamBuildPreProcessor` actually invokes it. If we would delete this line (intentionally or by mistake), no UT will break and nobody notices.